File: tests/integration/construct_component_slow/python/component.py
# Copyright 2016-2021, KhulnaSoft Ltd.  All rights reserved.

from typing import Any, Optional

import codeinfra

class Component(codeinfra.ComponentResource):
    def __init__(self, name: str, opts: Optional[codeinfra.ResourceOptions] = None):
        props = dict()
        super().__init__("testcomponent:index:Component", name, props, opts, True)

File: tests/integration/construct_component_slow/nodejs/component.ts
// Copyright 2016-2021, KhulnaSoft Ltd.  All rights reserved.

import * as codeinfra from "@codeinfra/codeinfra";

export class Component extends codeinfra.ComponentResource {
    constructor(name: string, opts?: codeinfra.ComponentResourceOptions) {
        super("testcomponent:index:Component", name, {}, opts, true);
    }
}
